torchtext/vocab.py

```diff
@@ -407,7 +407,7 @@ def __init__(self, name='840B', dim=300, **kwargs):
 
 class FastText(Vectors):
 
-    url_base = 'https://s3-us-west-1.amazonaws.com/fasttext-vectors/wiki.{}.vec'
+    url_base = "https://dl.fbaipublicfiles.com/fasttext/vectors-wiki/wiki.{}.vec"
```

Comment:
Could you use ' instead of " for consistency?

Reply:
sure, I changed it.